installer/compose/README.md

```diff
@@ -90,6 +91,18 @@ docker-compose -f docker-compose.full.yml down
 #docker-compose -f docker-compose.full.yml down
 ```
 
+We introduce quickstart deployment that comes with a set of predefined StreamPipes assets, to use the quickstart mode, just build the Docker image and start **quickstart** option:
+```bash
+docker-compose -f docker-compose.quickstart.yml build script-runner
+docker-compose -f docker-compose.quickstart.yml up -d
+# go to after all services are started http://localhost
+```
+Stopping the **quickstart** option:
+```bash
+docker-compose -f docker-compose.quickstart.yml down
+#docker-compose -f docker-compose.quickstart.yml down
```

Comment:
This comment is actually a duplicate right?

Reply:
Thanks for the suggestion, here actually I am trying to make the format is same as the "full" and "nats" option, I could delete the duplicate row. Should I delete the duplicate "full" and "nats" row together?

Reply:
Yes I think you can delete this row there as well
Thanks for pointing out!

Reply:
Hi, @bossenti ! 
I have refactored the README file, deleting the duplicate rows.